docs/source/rest-api.rst

```diff
@@ -1324,78 +1211,6 @@ Tag for a run.
 | value      | ``STRING`` | The tag value. |
 +------------+------------+----------------+
 
-Tag keys that start with ``mlflow.`` are reserved for internal use. The following tags are set
```

Comment:
Don't we want to keep this unfortunately-manually-written part?

Reply:
Moved: https://github.com/mlflow/mlflow/blob/master/docs/source/tracking.rst#id95

Reply:
nice - thanks!